doc/tutorial/bindings/python3/session.rst

```diff
@@ -0,0 +1,10 @@
+.. _bindings python3 creating and configuring a session:
+
+Creating and configuring a session
+==================================
+
```

Comment:
Again some introduction here. What we mean by the session, what actions can user do after setting up the session...